src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -25,11 +27,18 @@
 import reposense.model.RepoLocation;
 import reposense.parser.InvalidLocationException;
 import reposense.util.FileUtil;
+import reposense.util.TestUtil;
 
 /**
  * Contains templates for git testing.
  */
 public class GitTestTemplate {
+    protected static final Date BLAME_TEST_SINCE_DATE = TestUtil.getDate(2018, Calendar.FEBRUARY, 6);
+    protected static final Date BLAME_TEST_UNTIL_DATE = TestUtil.getEndOfDate(2018, Calendar.FEBRUARY, 8);
+    protected static final Date MOVED_FILE_SINCE_DATE = TestUtil.getDate(2018, Calendar.FEBRUARY, 7);
+    protected static final Date MOVED_FILE_UNTIL_DATE = TestUtil.getEndOfDate(2018, Calendar.FEBRUARY, 9);
+    protected static final Date PR_617_SINCE_DATE = TestUtil.getDate(2019, Calendar.MARCH, 28);
```

Comment:
Should rename it to its relevant name. Otherwise, we need to go to PR 617 to see what it is about.

Reply:
PR 617 is a one-line file used to test the author with emails that have the `+` operator within it, for e.g. `35282962+githubid@users.noreply.github.com`

I am unsure how to rename it, would EMAIL_WITH_ADDITION_TEST be alright?

Reply:
Acceptable.